source/change-streams/tests/unified/change-streams.yml

```diff
@@ -140,7 +140,50 @@ tests:
   - description: "Test that comment is set on getMore"
     runOnRequirements:
       - minServerVersion: "4.4.0"
-        topologies: [ single, replicaset ]
+        topologies: [ replicaset ]
```

Comment:
Should you add a comment explaining why sharded clusters are excluded here (e.g. potentially empty `getMore` responses)?

If we do want to include sharded clusters, I suppose we could utilize `ignoreExtraEvents`, which was added in https://github.com/mongodb/specifications/commit/ff0c705f9cc95a5a9c8c595dbf9384cb41dfcdd2 for [DRIVERS-1713](https://jira.mongodb.org/browse/DRIVERS-1713); however, that would then require drivers to implement those test runner changes just to sync these tests.

The best course of action may be to just leave the comment here and open another DRIVERS ticket to come back to these tests and add `ignoreExtraEvents` to allow sharded clusters. That wouldn't be something you'd need to work on yourself -- reporting the issue and leaving it in the backlog would be sufficient just so we don't lose track of it.

Reply:
Good point, thank you. Created a ticked and added corresponding comment to tests.